packages/lambda_streams/lambda_streams.0.1.2/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "0.1.2"
```

Comment:
```suggestion
```

Reply:
It’s just metadata, can be deleted here without the need to retag. Opam relies on this file, not the one on the source repository